### PR TITLE
page: Implement Transfer page

### PR DESCRIPTION
**Pull request by Khadeeejah**
Created a sub-menu route in the side-bar 
Created a new component called GetStartedCard2 to fit the content of the cards
 waiting for a feedback 

issue no #19

### CODE REVIEW

**Review by MeisterJustice — CHANGES_REQUESTED** [review]
Your deployment is failing

**Review by Frontend-io — COMMENTED** [review]
image

Your build is currently failing with this error

**Comment by Khadeeejah** [comment]
Thank you, I couldn't view the error. I'll look into uit

File: components/GetstartedCard2/data.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export interface CardWithImage {
+    title: string
+    desc: string
+    route: string
+}
+
+export const startIntegratingData: CardWithImage[] = [
```
[the remark this answers is not in the document]
Comment (Khadeeejah) [reply components/GetstartedCard2/data.ts]: @Frontend-io  please check

[the remark this answers is not in the document]
Comment (iamnotstatic) [reply components/GetstartedCard2/data.ts]: Hi @Frontend-io, Can you please look into this.

[the remark this answers is not in the document]
Comment (MeisterJustice) [reply components/GetstartedCard2/data.ts]: @Khadeeejah can you fix conflicts please?

File: components/GetstartedCard2/data.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export interface CardWithImage {
```
Comment (Frontend-io) [inline components/GetstartedCard2/data.ts:1]: let's take out this data.ts file. The content here is too little to create a file for. Please put the content on the jsx directly.

File: components/GetstartedCard2/index.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames'
```
Comment (Frontend-io) [inline components/GetstartedCard2/index.tsx:1]: GetstartedCard2 is too generic to use as a component name. Please rename.

File: components/GetstartedCard2/index.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+import LinkIcon from 'public/icons/link-icon'
+import ArrowRightIcon from 'public/icons/arrow-right-icon'
+
+interface MyProps {
+    desc: string,
+    title: string,
+    route: string,
+}
+
+// card with Icon component
```
Comment (Frontend-io) [inline components/GetstartedCard2/index.tsx:17]: There's no need to create two components, one with an icon and one without. Please use one component and conditionally render the icon if an icon prop is passed.

File: components/mdx-components.tsx [hunk 5]
```diff
@@ -48,6 +49,9 @@ export const components: Record<string, FC<Record<string, any>>> = {
   GetstartedCard() {
     return <GetstartedCard />
   },
+  GetstartedCard2() {
```
Comment (Frontend-io) [inline components/mdx-components.tsx:52]: We won't be needing GetStartedCard2. If there is a difference between this and the GetstartedCard, please conditionally render a version of the card or use [the early return ](https://javascript.plainenglish.io/early-return-with-react-hooks-f96fa4a33124)to return another variant of the card.

**Review by Frontend-io — CHANGES_REQUESTED** [review]

File: components/GetstartedCard2/index.tsx (hunk 4 above)
Comment (Khadeeejah) [reply components/GetstartedCard2/index.tsx:17]: fixed

**Review by MeisterJustice — APPROVED** [review]

**Pull request merged**